server/src/schema.graphql

```diff
@@ -4,12 +4,14 @@ type Query {
 
 type Mutation {
   createUser(data: CreateUserInput): SignupPayload!
+  verifyUserEmail(username: String!, token: String!): VerificationPayload!
```

Comment:
for one parameter do you need to create a separate input type? or are you planning to add more properties.

{ success: Boolean }

Reply:
I prefer this way as verification payload might be useful in future resolvers as well.